### PR TITLE
frontend role

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets:
- [#40790](https://edu-redmine.sun-asterisk.vn/issues/40790)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -37,4 +38,25 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function authenticated($request, $user)
+    {
+        if ($user[ 'role' ]) {
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/LoginController.php]:
```suggestion
        if ($user['role']) {
```

File: app/Http/Controllers/Auth/LoginController.php [hunk 2]
```diff
@@ -37,4 +38,25 @@ public function __construct()
...
+            return redirect(route('dashboard'));
+        }
+
+        return redirect(route('home.user', ['id' => $user[ 'id' ]]));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/LoginController.php]: tương tự

File: app/Http/Controllers/Auth/LoginController.php [hunk 3]
```diff
@@ -37,4 +38,25 @@ public function __construct()
...
+    }
+
+    protected function attemptLogin(Request $request)
+    {
+        $credentials = array_merge($this->credentials($request), ['is_active' => 1]);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Auth/LoginController.php]: đưa số vào config

**Comment by ghost** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**